pyproject.toml

```diff
@@ -21,6 +21,13 @@ metriq-client = { git = "https://github.com/unitaryfund/metriq-client.git", bran
 matplotlib = "^3.9.2"
 qiskit-experiments = "^0.7.0"
 
+[tool.poetry.group.dev.dependencies]
+ruff = "^0.7.3"
+pre-commit = "^4.0.1"
+
+[tool.ruff]
```

Comment:
Can we add:

```
line-length = 120
```

80 is annoyingly conservative

Reply:
120 is too long for me and others that use two windows side by side. PEP8 standard is 80, and Black's default is 88. I changed it to 100 in [60dc682](https://github.com/unitaryfund/metriq-gym/pull/22/commits/60dc6825e6d7c81ad2c01e866ecaf38de5ed6cc9)